exotic/exotic.py

```diff
@@ -713,43 +713,78 @@ def deg_to_pix(exp_ra, exp_dec, ra_list, dec_list):
     return np.unravel_index(dist.argmin(), dist.shape)
 
 
-# Check the ra and dec against the plate solution to see if the user entered in the correct values
-def check_targetpixelwcs(pixx, pixy, expra, expdec, ralist, declist):
+def check_target_pixel_wcs(input_x_pixel, input_y_pixel, info_dict, ra_list, dec_list, image_data, obs_time):
+    """
+    Verify the provided pixel coordinates match the target's right ascension and declination.
+    """
+    updated_ra, updated_dec = update_coordinates_with_proper_motion(info_dict, obs_time)
+
+    calculated_y_pixel, calculated_x_pixel = deg_to_pix(updated_ra, updated_dec, ra_list, dec_list)
+
+    centroid_x, centroid_y, sigma_x, sigma_y = get_psf_parameters(image_data, calculated_x_pixel, calculated_y_pixel)
+
+    return check_coordinates(input_x_pixel, input_y_pixel, centroid_x, centroid_y, sigma_x, sigma_y,
+                             calculated_x_pixel, calculated_y_pixel)
+
+
+def get_psf_parameters(image_data, x_pixel, y_pixel):
+    psf_data = fit_centroid(image_data, [x_pixel, y_pixel], 0)
+    return psf_data[0], psf_data[1], psf_data[3], psf_data[4]
+
+
+def check_coordinates(input_x_pixel, input_y_pixel, centroid_x, centroid_y, sigma_x, sigma_y,
+                      calculated_x_pixel, calculated_y_pixel):
     while True:
         try:
-            uncert = 1 / 36
```

Comment:
Not that this matters (eliminated code) but int divided by int is always float in Python 3. However, such expressions I believe benefit readability by explicitly declaring the nature of the numerics, e.g. 
> 1. / 36. preferred over 1 / 36